vendor/github.com/containerd/containerd/vendor.conf

```diff
@@ -1,7 +1,7 @@
 github.com/containerd/go-runc f271fa2021de855d4d918dbef83c5fe19db1bdd5
-github.com/containerd/console 9290d21dc56074581f619579c43d970b4514bc08
+github.com/containerd/console 5d1b48d6114b8c9666f0c8b916f871af97b0a761
```

Comment:
Perhaps we should update this one as well (could be separate from this PR https://github.com/containerd/console/compare/9290d21dc56074581f619579c43d970b4514bc08...5d1b48d6114b8c9666f0c8b916f871af97b0a761)

Reply:
The docker/cli 18.06 vendor.conf is already updated to the same git commit: https://github.com/docker/cli/blob/bb983ee75b2616f240ff6a4c6745813fbe2661df/vendor.conf#L94

Reply:
🤦‍♂️ ah, thanks, didn't check that one in the end